### PR TITLE
A few mainly stylistic changes to search

### PR DESCRIPTION
**Pull request by AmeeMorris**
Styles, missing props etc!

### CODE REVIEW

File: lib/Search/styles.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
+$top-bar-height: 95px;
```
Comment (kyleharper) [inline lib/Search/styles.scss]: Hmm this seems like a leak from a consuming app. Search is a standalone component in `gather-ui` is there a way we can do this in an abstract manner?

Comment (kyleharper) [reply lib/Search/styles.scss]: One idea would be to create a story using both `TopBar` and `Search` together and then join the 2 with a new element like `top-bar__search-dropdown`.?

Then apply the styles to that - it'll then be clear that in our library we expect the 2 to be used together where as right now search can only ever be used in the top bar reducing it's usability to a single case.

**Comment by AmeeMorris** [comment]
[[Search] Build UI components](https://trello.com/c/tnSC2dpb/3502-search-build-ui-components)

**Comment by AmeeMorris** [comment]
@kyleharper yep, good thinking, changes done :)

**Pull request merged**